quirks/password-rules.json

```diff
@@ -204,6 +207,9 @@
     "visa.com": {
         "password-rules": "minlength: 6; maxlength: 32;"
     },
+    "vivo.com.br": {
+        "password-rules": "maxlength: 6; max-consecutive: 3; allowed: [0123456789];"
```

Comment:
Equal to: `maxlength: 6; max-consecutive: 3; allowed: digit;`

Reply:
awesome. thanks :)